kernel/classes/eznodeviewfunctions.php

```diff
@@ -605,6 +605,12 @@ static public function contentViewGenerate( $file, $args )
             $validation
         );
 
+        // by setting cache_ttl to 0 in tpl code no cachfile should be generated
+        if ( isset( $result['no_cache'] ) and $result['no_cache'] )
```

Comment:
symbols based boolean operators must be used according to coding standards.  can you change the```and``` to ```&&```?